<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Vignette illustrating the usage of gscreend on simulated data • gscreend</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js" integrity="sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Vignette illustrating the usage of gscreend on simulated data">
<meta property="og:description" content="gscreend">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">gscreend</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.17.2</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/gscreend_simulated_data.html">Vignette illustrating the usage of gscreend on simulated data</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/imkeller/gscreend/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>Vignette illustrating the usage of gscreend on
simulated data</h1>
                        <h4 data-toc-skip class="author">Katharina
Imkeller</h4>
            
            <h4 data-toc-skip class="date">2024-03-13</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/imkeller/gscreend/blob/HEAD/vignettes/gscreend_simulated_data.Rmd" class="external-link"><code>vignettes/gscreend_simulated_data.Rmd</code></a></small>
      <div class="hidden name"><code>gscreend_simulated_data.Rmd</code></div>

    </div>

    
    
<div class="section level2">
<h2 id="introduction">Introduction<a class="anchor" aria-label="anchor" href="#introduction"></a>
</h2>
<p>Pooled CRISPR perturbations screens employ a library of guide RNAs
(gRNAs) that is transduced into a pool of cells with the aim to induce a
single genetic perturbation in each cell. The perturbation effect is
assessed by measuring the abundance of each gRNA after the screen
selection phase and comparing it to its abundance in the plasmid
library. The main goal of the following analysis is the detection of
essential genes, i.e. genes whose knockout reduces the cell fitness. The
package gscreend provides a method to rank genes based on count
tables.</p>
</div>
<div class="section level2">
<h2 id="gscreend-workflow">gscreend workflow<a class="anchor" aria-label="anchor" href="#gscreend-workflow"></a>
</h2>
<p>In order to identify essential genes starting from raw gRNA count
data, gscreend performs the following analysis steps:</p>
<ol style="list-style-type: decimal">
<li><p>Input of raw gRNA counts at T0 (sequencing of library) and T1 (at
the end of the screen). Normalization and calculation of log fold
changes.</p></li>
<li><p>Split log fold changes into intervals dependent on the initial
count at T0.</p></li>
<li><p>For every interval fit a skew-normal distribution to the data to
model the null hypothesis (via least quantile regression).</p></li>
<li><p>Based on the null model calculate p-values for every
gRNA.</p></li>
<li><p>Rank gRNAs according to p-value and perform robust ranking
aggregation to calculate p-values on gene level.</p></li>
<li><p>Perform quality control of data and statistical model.</p></li>
</ol>
</div>
<div class="section level2">
<h2 id="installation">Installation<a class="anchor" aria-label="anchor" href="#installation"></a>
</h2>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw">if</span><span class="op">(</span><span class="op">!</span><span class="fu"><a href="https://rdrr.io/r/base/ns-load.html" class="external-link">requireNamespace</a></span><span class="op">(</span><span class="st">"BiocManager"</span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span><span class="op">)</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"BiocManager"</span><span class="op">)</span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://bioconductor.github.io/BiocManager/reference/install.html" class="external-link">install</a></span><span class="op">(</span><span class="st">"gscreend"</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/imkeller/gscreend" class="external-link">gscreend</a></span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Warning in fun(libname, pkgname): Package 'gscreend' is deprecated and will be removed from Bioconductor</span></span>
<span><span class="co">##   version 3.20</span></span></code></pre>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://bioconductor.org/packages/SummarizedExperiment" class="external-link">SummarizedExperiment</a></span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="analysis-of-simulated-data-with-gscreend">Analysis of simulated data with gscreend<a class="anchor" aria-label="anchor" href="#analysis-of-simulated-data-with-gscreend"></a>
</h2>
<div class="section level3">
<h3 id="input-data-grna-counts">Input data: gRNA counts<a class="anchor" aria-label="anchor" href="#input-data-grna-counts"></a>
</h3>
<p>The simulated data used in this example has been generated using the
simulation method available at <a href="https://github.com/imkeller/simulate_pooled_screen" class="external-link uri">https://github.com/imkeller/simulate_pooled_screen</a></p>
<p>Raw count data consists of gRNA counts in the library sequencing and
different replicates after the screen proliferation phase. In order to
estimate the effect of a specific gRNA on cell fitness, the relative
abundances of the gRNA before and after the proliferation phase will be
compares</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">raw_counts</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/utils/read.table.html" class="external-link">read.table</a></span><span class="op">(</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"simulated_counts.txt"</span>, package <span class="op">=</span> <span class="st">"gscreend"</span><span class="op">)</span>,</span>
<span>    header<span class="op">=</span><span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div>
<p>Generate a summarized experiment from the count data. gscreend
currently uses SummarizedExperiment objects as an input format.</p>
<p>The count matrix contains raw gRNA counts. Each row represents one
gRNA, each column represents one sample (T0, T1, replicates, …).</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">counts_matrix</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/cbind.html" class="external-link">cbind</a></span><span class="op">(</span><span class="va">raw_counts</span><span class="op">$</span><span class="va">library0</span>, </span>
<span>                        <span class="va">raw_counts</span><span class="op">$</span><span class="va">R0_0</span>, </span>
<span>                        <span class="va">raw_counts</span><span class="op">$</span><span class="va">R1_0</span><span class="op">)</span></span>
<span></span>
<span><span class="va">rowData</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/data.frame.html" class="external-link">data.frame</a></span><span class="op">(</span>sgRNA_id <span class="op">=</span> <span class="va">raw_counts</span><span class="op">$</span><span class="va">sgrna_id</span>,</span>
<span>                    gene <span class="op">=</span> <span class="va">raw_counts</span><span class="op">$</span><span class="va">Gene</span><span class="op">)</span></span>
<span></span>
<span><span class="va">colData</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/data.frame.html" class="external-link">data.frame</a></span><span class="op">(</span>samplename <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"library"</span>, <span class="st">"R1"</span>, <span class="st">"R2"</span><span class="op">)</span>,</span>
<span>                    <span class="co"># timepoint naming convention: </span></span>
<span>                    <span class="co"># T0 -&gt; reference, </span></span>
<span>                    <span class="co"># T1 -&gt; after proliferation</span></span>
<span>                    timepoint <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"T0"</span>, <span class="st">"T1"</span>, <span class="st">"T1"</span><span class="op">)</span><span class="op">)</span></span>
<span></span>
<span><span class="va">se</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/RangedSummarizedExperiment-class.html" class="external-link">SummarizedExperiment</a></span><span class="op">(</span>assays<span class="op">=</span><span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span>counts<span class="op">=</span><span class="va">counts_matrix</span><span class="op">)</span>,</span>
<span>                    rowData<span class="op">=</span><span class="va">rowData</span>, colData<span class="op">=</span><span class="va">colData</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="run-gscreend">Run gscreend<a class="anchor" aria-label="anchor" href="#run-gscreend"></a>
</h3>
<p>In this step a gscreend experiment object is generated that will
after the analysis contain all data related to gRNAs, genes and model
parameters.</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">pse</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/createPoolScreenExp.html">createPoolScreenExp</a></span><span class="op">(</span><span class="va">se</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Creating PoolScreenExp object from a SummarizedExperiment object.</span></span></code></pre>
<pre><code><span><span class="co">## References and samples are named correctly.</span></span></code></pre>
<pre><code><span><span class="co">## Data concerning sgRNA and genes is provided.</span></span></code></pre>
<p>Run gscreend with default parameters.</p>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">pse_an</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/RunGscreend.html">RunGscreend</a></span><span class="op">(</span><span class="va">pse</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Size normalized count data.</span></span></code></pre>
<pre><code><span><span class="co">## Calculated LFC.</span></span></code></pre>
<pre><code><span><span class="co">## Fitted null distribution.</span></span></code></pre>
<pre><code><span><span class="co">## Calculated p-values at gRNA level.</span></span></code></pre>
<pre><code><span><span class="co">## Ranking genes...</span></span></code></pre>
<pre><code><span><span class="co">## ... for positive fold changes</span></span></code></pre>
<pre><code><span><span class="co">## ... for negative fold changes</span></span></code></pre>
<pre><code><span><span class="co">## gscreend analysis has been completed.</span></span></code></pre>
</div>
<div class="section level3">
<h3 id="quality-control">Quality control<a class="anchor" aria-label="anchor" href="#quality-control"></a>
</h3>
<p>gscreend provides basic quality control functions for inspection of
replicate correlation for example.</p>
<div class="sourceCode" id="cb20"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="../reference/plotReplicateCorrelation.html">plotReplicateCorrelation</a></span><span class="op">(</span><span class="va">pse_an</span><span class="op">)</span></span></code></pre></div>
<p><img src="gscreend_simulated_data_files/figure-html/unnamed-chunk-7-1.png" width="384"></p>
<p>The <code><a href="../reference/plotModelParameters.html">plotModelParameters()</a></code> function can be used to
inspect the values of the parameters estimated for the skew normal
distribution of the logarithmic fold change data.</p>
<div class="sourceCode" id="cb21"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="../reference/plotModelParameters.html">plotModelParameters</a></span><span class="op">(</span><span class="va">pse_an</span><span class="op">)</span> </span></code></pre></div>
<p><img src="gscreend_simulated_data_files/figure-html/unnamed-chunk-8-1.png" width="700"></p>
</div>
</div>
<div class="section level2">
<h2 id="results">Results<a class="anchor" aria-label="anchor" href="#results"></a>
</h2>
<p>The ResultsTable function can be used to extract a table listing for
each gene the p-value and fdr. These values correspond to the results
from the statistical test indicting whether upon perturbation a specific
gene reduces (direction = “negative”), or increasing (direction =
“positive”) cell viability.</p>
<div class="sourceCode" id="cb22"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">res</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/ResultsTable.html">ResultsTable</a></span><span class="op">(</span><span class="va">pse_an</span>, direction <span class="op">=</span> <span class="st">"negative"</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="va">res</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##                          Name          fdr    pval       lfc</span></span>
<span><span class="co">## essential_0       essential_0 0.0004391101 0.00003 -1.421372</span></span>
<span><span class="co">## essential_1       essential_1 0.0000000000 0.00000 -1.984253</span></span>
<span><span class="co">## essential_10     essential_10 0.0001604621 0.00001 -1.605037</span></span>
<span><span class="co">## essential_100   essential_100 0.0000000000 0.00000 -3.075151</span></span>
<span><span class="co">## essential_1000 essential_1000 0.0000000000 0.00000 -2.023614</span></span>
<span><span class="co">## essential_1001 essential_1001 0.0538116592 0.00480 -1.032688</span></span></code></pre>
</div>
<div class="section level2">
<h2 id="session-info">Session Info<a class="anchor" aria-label="anchor" href="#session-info"></a>
</h2>
<div class="sourceCode" id="cb24"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html" class="external-link">sessionInfo</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## R Under development (unstable) (2024-03-12 r86109)</span></span>
<span><span class="co">## Platform: x86_64-apple-darwin20</span></span>
<span><span class="co">## Running under: macOS Monterey 12.7.3</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Matrix products: default</span></span>
<span><span class="co">## BLAS:   /Library/Frameworks/R.framework/Versions/4.4-x86_64/Resources/lib/libRblas.0.dylib </span></span>
<span><span class="co">## LAPACK: /Library/Frameworks/R.framework/Versions/4.4-x86_64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.0</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## locale:</span></span>
<span><span class="co">## [1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## time zone: UTC</span></span>
<span><span class="co">## tzcode source: internal</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## attached base packages:</span></span>
<span><span class="co">## [1] stats4    stats     graphics  grDevices utils     datasets  methods  </span></span>
<span><span class="co">## [8] base     </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## other attached packages:</span></span>
<span><span class="co">##  [1] SummarizedExperiment_1.33.3 Biobase_2.63.0             </span></span>
<span><span class="co">##  [3] GenomicRanges_1.55.3        GenomeInfoDb_1.39.8        </span></span>
<span><span class="co">##  [5] IRanges_2.37.1              S4Vectors_0.41.4           </span></span>
<span><span class="co">##  [7] BiocGenerics_0.49.1         MatrixGenerics_1.15.0      </span></span>
<span><span class="co">##  [9] matrixStats_1.2.0           gscreend_1.17.2            </span></span>
<span><span class="co">## [11] BiocStyle_2.31.0           </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## loaded via a namespace (and not attached):</span></span>
<span><span class="co">##  [1] sass_0.4.8              SparseArray_1.3.4       bitops_1.0-7           </span></span>
<span><span class="co">##  [4] fGarch_4032.91          lattice_0.22-5          digest_0.6.35          </span></span>
<span><span class="co">##  [7] magrittr_2.0.3          evaluate_0.23           grid_4.4.0             </span></span>
<span><span class="co">## [10] bookdown_0.38           fastmap_1.1.1           jsonlite_1.8.8         </span></span>
<span><span class="co">## [13] Matrix_1.6-5            BiocManager_1.30.22     purrr_1.0.2            </span></span>
<span><span class="co">## [16] gbutils_0.5             codetools_0.2-19        textshaping_0.3.7      </span></span>
<span><span class="co">## [19] jquerylib_0.1.4         Rdpack_2.6              timeSeries_4032.109    </span></span>
<span><span class="co">## [22] abind_1.4-5             cli_3.6.2               rlang_1.1.3            </span></span>
<span><span class="co">## [25] crayon_1.5.2            rbibutils_2.2.16        XVector_0.43.1         </span></span>
<span><span class="co">## [28] cachem_1.0.8            DelayedArray_0.29.9     yaml_2.3.8             </span></span>
<span><span class="co">## [31] cvar_0.5                S4Arrays_1.3.6          parallel_4.4.0         </span></span>
<span><span class="co">## [34] tools_4.4.0             BiocParallel_1.37.1     nloptr_2.0.3           </span></span>
<span><span class="co">## [37] memoise_2.0.1           GenomeInfoDbData_1.2.11 vctrs_0.6.5            </span></span>
<span><span class="co">## [40] R6_2.5.1                lifecycle_1.0.4         zlibbioc_1.49.0        </span></span>
<span><span class="co">## [43] fs_1.6.3                ragg_1.2.7              desc_1.4.3             </span></span>
<span><span class="co">## [46] pkgdown_2.0.7.9000      bslib_0.6.1             systemfonts_1.0.6      </span></span>
<span><span class="co">## [49] highr_0.10              xfun_0.42               knitr_1.45             </span></span>
<span><span class="co">## [52] spatial_7.3-17          fBasics_4032.96         htmltools_0.5.7        </span></span>
<span><span class="co">## [55] rmarkdown_2.26          timeDate_4032.109       compiler_4.4.0         </span></span>
<span><span class="co">## [58] RCurl_1.98-1.14</span></span></code></pre>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Katharina Imkeller, Wolfgang Huber.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.9000.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
